src/vse_sync_pp/analyze.py

```diff
@@ -8,21 +8,30 @@
 
 from .parse import PARSERS
 
-from .analyzers import ppsdpll
+from .analyzers.analyzer import Config
+from .analyzers import (
+    ppsdpll,
+    ts2phc,
+)
 
 ANALYZERS = {
     cls.id_: cls for cls in (
         ppsdpll.PhaseOffsetTimeErrorAnalyzer,
+        ts2phc.TimeErrorAnalyzer,
```

Comment:
if we go with  `ts2phc.TimeErrorAnalyzer` then rename `ppsdpll.PhaseOffsetTimeErrorAnalyzer` to  `ppsdpll.TimeErrorAnalyzer`